egui/src/widgets/image.rs

```diff
@@ -75,6 +78,17 @@ impl Image {
         self.sense = sense;
         self
     }
+
+    /// Rotate the image about an origin by some angle
+    ///
+    /// Positive angle is clockwise.
+    /// Origin is a vector in UV space (normalized space by default unless set by [`Image::uv`])
+    ///
+    /// To rotate about the center you can pass `Vec2::splat(0.5)` as the origin.
+    pub fn rotate(mut self, angle: f32, origin: Vec2) -> Self {
+        self.rotation = Some((Rot2::from_angle(angle), origin / self.uv.size()));
```

Comment:
dividing by uv.size doesn't make much sense to me – you need to take the whole uv rect into account.